packages/shared/src/badges/Badge.scss

```
@@ -1,10 +1,10 @@
 .ocs-preview-badge {
   &.pf-c-label {
```

now we will have to override PF5 classes, I guess...
```suggestion
  &.pf-v5-c-label {
```